.github/workflows/autobuild.yml

```diff
@@ -134,6 +134,12 @@ jobs:
 
     runs-on:                        ${{ matrix.config.building_on_os }}
     steps:
+      - name:                       Select Xcode version for Mac
+        if:                         ${{ matrix.config.target_os == 'macos' }}
+        uses:                       maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: '11.7'
```

Comment:
I'm usually not a fan to add comments in the style of "This is here because of issue #0123", but maybe just add a comment similar to the PR description to explain why the version is locked at all and why it is locked to exactly this version? This could help our future selves to figure out when it is safe to update. ;o

Reply:
Done